src/syntax.rs

```diff
@@ -18,6 +18,13 @@ pub enum Op2 {
     Eq
 }
 
+#[derive(Debug, PartialEq)]
+pub enum Printable {
+    Id(String),
+    Val(Val),
+    Seq(Val, Val),
```

Comment:
This is just a pair of values. Did you mean to write `Seq(Vec<Printable>)` or something to that effect?

Reply:
That's probably a bad name for it. Maybe `Array` or `Range` would be better? The eval evaluates `print(seq(ptr, len));` to something like
```
for i in range(ptr, ptr + len) {
  println!("{}; ", i*);
}
```

It's not exactly in the spirit of the language, but I found it useful to debug.

Reply:
Ah, very nice. Thanks! Would you rename it to Array or Range? I'm happy to merge this in.

Reply:
I pushed the changes. Glad you're happy with it.